### ISSUE TITLE
Dashboards fail to load in iOS 15 (2023.11.0b0)

### ISSUE THREAD

**Issue by RobertD502** [home-assistant/frontend#18419]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

After updating to 2023.11.0b0, I found that dashboards don't load when accessing Home Assistant with Safari(iOS 15.1) or the Home Assistant companion app (I'm assuming this is due to it also using webkit). The following is seen when either accessing Home Assistant via Mobile Safari or the iOS companion app:

![IMG_6627DFC1A169-1](https://github.com/home-assistant/frontend/assets/52541649/16fb70d8-c2c7-4f7e-8fa6-442a1c837eca)

I can confirm that this doesn't affect any other browsers (including more up to date Mac and iOS devices).
I have included an output from the dev console showing errors encountered on the iOS device.

### Describe the behavior you expected

Dashboards should load normally.

### Steps to reproduce the issue

Connect to Home Assistant on an iOS device running iOS 15.

### What version of Home Assistant Core has the issue?

2023.11.0b0

### What was the last working version of Home Assistant Core?

2023.10.5

### In which browser are you experiencing the issue with?

iOS Safari

### Which operating system are you using to run this browser?

iOS 15.1

### Javascript errors shown in your browser console/inspector

```txt
[Error] Unhandled Promise Rejection: SyntaxError: Invalid regular expression: invalid group specifier name
	promiseEmptyOnRejected (3143-FKwm94Gin5A.js:1)
	promiseReactionJob
```
```
[Error] Error loading page – "lovelace"
SyntaxError: Invalid regular expression: invalid group specifier name
parseModule
(anonymous function)
asyncFunctionResume
(anonymous function)
promiseReactionJobWithoutPromise
	(anonymous function) (app-wENyGOpR21s.js:2:88631)
	promiseReactionJob
```
```


### Additional information

![image](https://github.com/home-assistant/frontend/assets/52541649/b3d62bed-ad2e-4075-9238-aebf1727a40b)

**Comment by piitaya** [comment]
`2023.11.0b1` will provide a fix for this one https://github.com/home-assistant/frontend/issues/18418. I hope it will resolve your issue too.

**Comment by RobertD502** [comment]
Without me going and hunting it down, do you happen to know where the frontend files are located within Home Assistant OS or the container within the OS(I have SSH access to the host via port 2222, so I can make the changes seen in the PR to test if it is related instead of waiting on the beta 1 build)?

**Comment by bramkragten** [comment]
Have you tried if this also happens in safe mode?

**Comment by bramkragten** [comment]
This might be related to https://github.com/home-assistant/frontend/pull/18297

**Comment by bramkragten** [comment]
Or https://github.com/home-assistant/frontend/pull/18250

**Comment by RobertD502** [comment]
> Have you tried if this also happens in safe mode?

Same issue in safe mode. Regardless of the dashboard, it happens - Energy page, Settings Page, etc. None of them load, but I'll have to wait for b1 to see if the previously mentioned fix does the trick.

**Comment by Rrapa** [comment]
2023.11.0b1 is crashing dashboards on iOS 15 again  I'm afraid ...

**Comment by silamon** [comment]
The b1 build didn't include the fix for this issue. I see it's now picked up, and this fix will be included in b2.

**Comment by RobertD502** [comment]
> The b1 build didn't include the fix for this issue. I see it's now picked up, and this fix will be included in b2.

I'm not sure what fix you're referring to. The potential fix mentioned in piitaya's comment was part of beta1 (the PR was part of the `frontend 20231026.0` release, which was included in the `core 2023.11.0b1` release.)

**Comment by silamon** [comment]
What piitaya mentioned was a fix for a different issue which was most likely related. Bram included a different [fix](https://github.com/home-assistant/frontend/pull/18438) which closed this issue. That fix is in todays frontend release 20231027.0 which will end up in b2.

**Comment by RobertD502** [comment]
Confirmed working in beta2.

**Comment by Rrapa** [comment]
Same here - confirmed working

**Comment by madebywitteveen** [comment]
Same issue here; how to install B2 ?

**Comment by RobertD502** [comment]
> Same issue here; how to install B2 ?

No need to install the beta. Just install 2023.11.1, which includes all of the fixes done in the 2023.11 beta.

**Comment by madebywitteveen** [comment]
Thanks! That and restarting the appdeamon did the trick 
-------- Original message --------From: "Dr. Drinovac" ***@***.***> Date: 04/11/2023  20:59  (GMT+01:00) To: home-assistant/frontend ***@***.***> Cc: madebywitteveen ***@***.***>, Comment ***@***.***> Subject: Re: [home-assistant/frontend] Dashboards fail to load in iOS 15 (2023.11.0b0) (Issue #18419) 

Same issue here; how to install B2 ?

No need to install the beta. Just install 2023.11.1, which includes all of the fixes done in the 2023.11 beta.
